### PR TITLE
[NOC-385]: Changes for Acceptance criteria #4 (NOC 2021 filters in r…

### PR DESCRIPTION
**Pull request by sunanditasahu**

### CODE REVIEW

File: src/WorkBC.Web/Services/RecommendedJobsService.cs [hunk 1]
```diff
@@ -166,6 +167,13 @@ public async Task<AccountCriteria> GetUserAccountCriteria(JobSeeker jobSeeker)
                 .OrderByDescending(a => a.Count)
                 .ToDictionary(k => k.Term, v => v.Count);
 
+            // group the saved jobs by noc code 2021
```
Comment (infojunkie) [inline src/WorkBC.Web/Services/RecommendedJobsService.cs:170]: Looks good, but do we need to keep the processing for `NocCodes` as well? right above this block.

**Review by infojunkie — COMMENTED** [review]
I have a single question before approving

File: src/WorkBC.Web/Services/RecommendedJobsService.cs (hunk 1 above)
Comment (sunanditasahu) [reply src/WorkBC.Web/Services/RecommendedJobsService.cs:170]: No, we don't need to, but we have taken the approach everywhere else to add Noc 2021 code logic and leave Noc 2016 as is. I can remove and test it if you approve.

Comment (infojunkie) [reply src/WorkBC.Web/Services/RecommendedJobsService.cs:170]: Yes, but in this case we're doing an additional query for the NOC 2016 which we're no longer using... maybe best to comment out the query and ensure the code doesn't break.

Comment (sunanditasahu) [reply src/WorkBC.Web/Services/RecommendedJobsService.cs:170]: The UI display breaks when I comment out the Noc Code query part. I am investigating the part where the UI might be dependent on the Noc code data.

Comment (infojunkie) [reply src/WorkBC.Web/Services/RecommendedJobsService.cs:170]: Don't spend too much time on this - it's more important to solve actual bugs :-D

**Comment by sunanditasahu** [comment]
Closing this PR as the changes have been moved to another PR https://github.com/bcgov/workbc-jb/pull/162

**Pull request closed without merging**